### ISSUE TITLE
error on install when using pipenv

### ISSUE THREAD

**Issue by marafa** [bluenote10/tabloo#4]
```
[root@fc30 tabloo]# pipenv install --dev tabloo
Creating a virtualenv for this project…
Pipfile: /root/tabloo/Pipfile
Using /usr/bin/python3.7 (3.7.4) to create virtualenv…
⠇ Creating virtual environment...Using base prefix '/usr'
New python executable in /root/.local/share/virtualenvs/tabloo-yxMSgON9/bin/python3.7
Also creating executable in /root/.local/share/virtualenvs/tabloo-yxMSgON9/bin/python
Installing setuptools, pip, wheel...done.
Running virtualenv with interpreter /usr/bin/python3.7

✔ Successfully created virtual environment! 
Virtualenv location: /root/.local/share/virtualenvs/tabloo-yxMSgON9
Installing tabloo…
Adding tabloo to Pipfile's [dev-packages]…
✔ Installation Succeeded 
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
✔ Success! 
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock (bed5cc)!
Installing dependencies from Pipfile.lock (bed5cc)…
An error occurred while installing tabloo==0.0.5 --hash=sha256:756891169475ba39db2ed9446482c4223640d7409c7b06eca2b0e597875ecdce --hash=sha256:85971b0ca68a3fd473148f6e8ac38af7e66ceb01b432de61c7b2df848393dd7f --hash=sha256:756891169475ba39db2ed9446482c4223640d7409c7b06eca2b0e597875ecdce --hash=sha256:85971b0ca68a3fd473148f6e8ac38af7e66ceb01b432de61c7b2df848393dd7f! Will try again.
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 13/13 — 00:00:20
Installing initially failed dependencies…
[pipenv.exceptions.InstallError]: ['Collecting tabloo==0.0.5 (from -r /tmp/pipenv-eg3n439t-requirements/pipenv-7bbj74nv-requirement.txt (line 1))', '  Using cached https://files.pythonhosted.org/packages/bb/75/9c7008af9a7e3f006872d8ee3c3421c1fe0caff2f196b746f5b88924775e/tabloo-0.0.5.tar.gz']
[pipenv.exceptions.InstallError]: ['ERROR: Command errored out with exit status 1:', '     command: /root/.local/share/virtualenvs/tabloo-yxMSgON9/bin/python3.7 -c \'import sys, setuptools, tokenize; sys.argv[0] = \'"\'"\'/tmp/pip-install-nooms985/tabloo/setup.py\'"\'"\'; __file__=\'"\'"\'/tmp/pip-install-nooms985/tabloo/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' egg_info --egg-base pip-egg-info', '         cwd: /tmp/pip-install-nooms985/tabloo/', '    Complete output (5 lines):', '    Traceback (most recent call last):', '      File "<string>", line 1, in <module>', '      File "/tmp/pip-install-nooms985/tabloo/setup.py", line 3, in <module>', '        with open("requirements.txt") as f:', "    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'", '    ----------------------------------------', 'ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.']
ERROR: ERROR: Package installation failed...
  ☤  ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/1 — 0
```

**Comment by marafa** [comment]
this happens when using `pipenv` . when using `pip` - title updated to reflect more accurately

**Comment by bluenote10** [comment]
Does the installation work for you using just `pip`? I don't use `pipenv`, is it possible to get more information on the error?

**Comment by bluenote10** [comment]
I just tried it myself and it seems to work for me:

![image](https://user-images.githubusercontent.com/3620703/62330580-5afd2600-b4b9-11e9-85ab-22fdd23b5b4e.png)

I'd assume it's an issue in pipenv then. Did you try with this `pipenv` version?

**Comment by marafa** [comment]
yes. double checked. same pipenv version
also tried in a fresh directory in /tmp - same result

i will branch out and check  if my OS/fedora has something up with pipenv

thanks

**Comment by 411an** [comment]
```
c:\Python37\Scripts>pip install tabloo
Collecting tabloo
  Using cached https://files.pythonhosted.org/packages/f5/5a/a65d0d45ae55b9ceb318c312e093970f6d77b8ece78a189c7607fd0a4b6f/tabloo-0.0.10.tar.gz
    ERROR: Command errored out with exit status 1:
     command: 'c:\python37\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\{profile name}\\AppData\\Local\\Temp\\pip-install-ubm0jouu\\tabloo\\setup.py'"'"'; __file__='"'"'C:\\Users\\{profile name}\\AppData\\Local\\Temp\\pip-install-ubm0jouu\\tabloo\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: C:\Users\{profile name}\AppData\Local\Temp\pip-install-ubm0jouu\tabloo\
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\{profile name}\AppData\Local\Temp\pip-install-ubm0jouu\tabloo\setup.py", line 3, in <module>
        with open("requirements.txt") as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by bluenote10** [comment]
@411an Thanks for the report that helped me to understand what is going on. I couldn't reproduce the problem initially, because under Python 2.7, pip picks the wheel instead of the source distribution, which was working fine. By forcing to use the source distribution I could reproduce the problem now, and I think I have found the issue. I've uploaded a new version now, I hope that resolved the issue.
